internal/tests/api/credentials/credentials_test.go

```diff
@@ -500,7 +500,7 @@ func TestUpdateAfterKeyRotation(t *testing.T) {
 	for {
 		jobs, err := scopesClient.ListKeyVersionDestructionJobs(ctx, proj.PublicId)
 		require.NoError(err)
-		if len(jobs.Items) == 1 {
+		if len(jobs.Items) >= 1 {
```

Comment:
![image](https://github.com/hashicorp/boundary/assets/6604151/3b187d14-0bdd-46d0-b702-8fda7bf46bf2)

Is this the correct fix? I think I wrote this flaky test 😛.

Reply:
We want to ensure that rotation happens right? If the test is slow and rotation happens more than once, the test fails, but the actual behavior is success.